internal/test/builder/api/builder_service_account_xrhid.go

```diff
@@ -0,0 +1,94 @@
+package api
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/podengo-project/idmsvc-backend/internal/test/builder/helper"
+	builder_helper "github.com/podengo-project/idmsvc-backend/internal/test/builder/helper"
+	identity "github.com/redhatinsights/platform-go-middlewares/v2/identity"
+)
+
+type ServiceAcccountXRHID interface {
+	Build() identity.XRHID
+	WithAccountNumber(value string) ServiceAcccountXRHID
+	WithEmployeeAccountNumber(value string) ServiceAcccountXRHID
+	WithOrgID(value string) ServiceAcccountXRHID
+	WithAuthType(value string) ServiceAcccountXRHID
+
+	WithClientID(value string) ServiceAcccountXRHID
+	WithUsername(value string) ServiceAcccountXRHID
+}
+
+type serviceAcccountXRHID identity.XRHID
+
+// NewServiceAccountXRHID create a new builder to get a XRHID for a user
+func NewServiceAccountXRHID() ServiceAcccountXRHID {
+	var orgID = strconv.Itoa(int(builder_helper.GenRandNum(1, 100000)))
+	// See: https://github.com/coderbydesign/identity-schemas/blob/add-validator/3scale/schema.json
```

Comment:
The link doesn't work. Should it be https://github.com/RedHatInsights/identity-schemas/blob/main/3scale/identities/service-account.json as in the previous PR?

Reply:
Thanks! you are right! updated!